conduit/plugins/importers/syncmode/syncmode_importer.go

```diff
@@ -0,0 +1,168 @@
+package syncmode
```

Comment:
Should we make the package something like the importer name? `syncalgod` or something like that. I think that "syncmode" could be confusing as a name without the algod context.

Reply:
Good idea.  Done.